config/initializers/action_cable_patch.rb

```diff
@@ -0,0 +1,45 @@
+ActiveSupport.on_load(:action_cable) do
```

Comment:
@skateman What changed here over what I helped you with before?  I'm wondering what we were missing before...